ONSdigital/response-operations-ui
# Update CE page to allow multimode

# What and why?
This PR enables multimode in CE. I have taken the opportunity to remove the logic from the frontend and move it into the code for ce-ci-section.html where realistically it should be. Although I would have liked to moved build_collection_exercise_details and the new ci table context builder I have left where it is at the moment, it shouldn't really be in the view. I have made the code more explicit (eq/seft function called as such) and efficient (i.e not calling collection instrument on every pass; moving eq_ci_selectors; refactoring test etc) where it has been on my route. I have also moved redundancy for the same reason 

N.B 

The test change tab is minimised due to the amount of changes, rather than deleted as it might appear on the files changed section

Ideally ce-collection-instrument-section.html should be refactored as well, but time restrictions meant I have left mainly as if and changed to be controlled from the URL. We should aim for a clear logic/display separation where possible. 

# How to test?
Whether on GCP or local you will need this branch as well https://github.com/ONSdigital/ras-collection-instrument/pull/295 
Initially SEFT/EQ should be tested separately to make sure they still work as normal. Once that works you need to add in a EQ_AND_SEFT survey, this can be done by updating the survey_mode (survey scheme) in the DB or setting the env MULTI_MODE_ENABLED=True allowing the creation/edit of a new EQ_AND_SEFT survey on the UI. Once that is done create a CE and make sure both eQ and SEFT CI options are available and also work as expected.

N.B This doesn't match the image on the card as I haven't done the total CI count, I missed it initially and decided I needed some clarification on when it should appear, with that in mind and down to the blocking nature of this card I have split it out 

# Trello
